frontend/js/require.config.js

```diff
@@ -35,6 +36,11 @@ require.config({
     waitSeconds: 10,
 
     shim: {
+        "hls": {
+            exports: "hls",
+            deps: ['require']
+            },
```

Comment:
Sorry to be nitpicky like this, but this `}` is still indented too far. This configuration block should look like all the others around it.

Reply:
Solved because I deleted this part of the code.